main.go

```diff
@@ -272,6 +272,7 @@ func visit(url *url.URL) {
 			ServerName:         host,
 			InsecureSkipVerify: insecure,
 			Certificates:       readClientCert(clientCertFile),
+			MinVersion:         tls.VersionTLS10,
```

Comment:
Let's make this simpler and set the min TLS version to 1.2 then we only have to worry about 1.2 vs 1.3.

Reply:
Done

Reply:
Any updates? Do I need anything else to do?